engagement_database/data_models.py

```diff
@@ -68,6 +69,12 @@ def __init__(self, text, timestamp, participant_uuid, direction, channel_operato
         self.coda_id = coda_id
         self.last_updated = last_updated
 
+    def get_latest_labels(self):
+        """
+        Returns the latest label assigned to each code scheme.
```

Comment:
```suggestion
        Returns the latest label assigned to each message.
```

Reply:
It's definitely one per code scheme, as per core_data_modules.data_models.message.get_latest_labels